Signal/src/AppDelegate.m

```diff
@@ -793,6 +793,9 @@ - (void)databaseViewRegistrationComplete
     [AppVersion.instance appLaunchDidComplete];
 
     [self ensureRootViewController];
+
+    // If there were any messages in our local queue which we hadn't yet processed.
+    [[OWSMessageReceiver sharedInstance] handleAnyUnprocessedEnvelopesAsync];
```

Comment:
FYI we could observe kNSNotificationName_DatabaseViewRegistrationComplete but this is fine too.